README.md

```diff
@@ -126,8 +126,55 @@ Within graphql (the language you query information from and that this plugin put
 
 Keys can be found in Airtable by clicking `Help > API Documentation`.
 
-The API key can be specified in `gatsby-config.js` as noted in the previous section-- **this exposes your key to anyone viewing your repository and is not recommended**.
+The API key can be specified in `gatsby-config.js` as noted in the previous section-- **this exposes your key to anyone viewing your repository and is not recommended. You should inject your API key as recommended below to prevent it from being committed to source control**.
```

Comment:
```suggestion
The API key can be hard coded directly in `gatsby-config.js` as noted in the previous section-- **this exposes your key to anyone viewing your repository and is not recommended. You should inject your API key as recommended below to prevent it from being committed to source control**.
```